merged_schema/HPXMLMerged.xsd

```diff
@@ -2216,6 +2216,110 @@
+															<xs:element minOccurs="0" name="FractionChargedAtHome" type="Fraction">
+																<xs:annotation>
+																	<xs:documentation>The fraction of charging energy provided by an at-home charger.</xs:documentation>
+																</xs:annotation>
+															</xs:element>
+															<xs:element minOccurs="0" name="ConnectedCharger" type="LocalReference"/>
+														</xs:sequence>
+													</xs:complexType>
+												</xs:element>
+												<xs:element minOccurs="0" name="PHEV">
+													<xs:annotation>
+														<xs:documentation>Plug-in hybrid electric vehicle</xs:documentation>
+													</xs:annotation>
+													<xs:complexType>
+														<xs:sequence>
+															<xs:element minOccurs="0" name="Battery" type="VehicleBattery"/>
+															<xs:element minOccurs="0" name="FractionChargedAtHome" type="Fraction">
+																<xs:annotation>
+																	<xs:documentation>The fraction of charging energy provided by an at-home charger.</xs:documentation>
+																</xs:annotation>
+															</xs:element>
+															<xs:element minOccurs="0" name="ConnectedCharger" type="LocalReference"/>
+															<xs:element minOccurs="0" name="FuelType" type="VehicleFuelType"/>
+														</xs:sequence>
+													</xs:complexType>
+												</xs:element>
+												<xs:element minOccurs="0" name="HEV">
+													<xs:annotation>
+														<xs:documentation>Hybrid electric vehicle</xs:documentation>
+													</xs:annotation>
+													<xs:complexType>
+														<xs:sequence>
+															<xs:element minOccurs="0" name="FuelType" type="VehicleFuelType"/>
+														</xs:sequence>
+													</xs:complexType>
+												</xs:element>
+												<xs:element minOccurs="0" name="ICE">
+													<xs:annotation>
+														<xs:documentation>Internal combustion engine</xs:documentation>
+													</xs:annotation>
+													<xs:complexType>
+														<xs:sequence>
+															<xs:element minOccurs="0" name="FuelType" type="VehicleFuelType"/>
+														</xs:sequence>
+													</xs:complexType>
+												</xs:element>
+												<xs:element minOccurs="0" name="Other">
+													<xs:complexType>
+														<xs:sequence>
+															<xs:element minOccurs="0" name="FuelType" type="VehicleFuelType"/>
+														</xs:sequence>
+													</xs:complexType>
+												</xs:element>
+											</xs:choice>
+										</xs:complexType>
+									</xs:element>
+									<xs:element minOccurs="0" name="PrimaryLocation" type="VehicleLocation"/>
+									<xs:element minOccurs="0" name="MilesDrivenPerYear" type="HPXMLDoubleGreaterThanZero"/>
+									<xs:element minOccurs="0" name="HoursDrivenPerWeek" type="HPXMLDoubleGreaterThanOrEqualToZero"/>
+									<xs:element minOccurs="0" name="FuelEconomy">
```

Comment:
Maybe we should allow more than one of these like we do for heating and cooling efficiency in case there are more than one way to describe this for the same car. 

Is this the EPA rating? I would encourage you to look at the [EPA fuel economy label](https://www.epa.gov/greenvehicles/learn-about-fuel-economy-label) and have those fields in here.